contracts/multisig/src/msg.rs

```diff
@@ -22,7 +22,11 @@ pub enum ExecuteMsg {
     KeyGen {
         key_id: String,
         snapshot: Snapshot,
-        pub_keys: HashMap<String, HexBinary>,
+        pub_keys: HashMap<String, (KeyType, HexBinary)>,
+    },
+    RegisterPublicKey {
+        public_key: HexBinary,
+        key_type: KeyType,
```

Comment:
why not use the `PublicKey` enum instead of the type + hex tuple?

Reply:
Agree with Chris